### PR TITLE
Enabled additional models in CI

### PR DESCRIPTION
**Pull request by MohitIntel**
Enabled CodeLlama finetune(13b) and inference(34b), StableLM and T5-small inference models in CI.

### CODE REVIEW

File: tests/baselines/CodeLlama_13b_Instruct_hf.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "gaudi2": {
+        "wikitext": {
+            "num_train_epochs": 2,
```
Comment (libinta) [inline tests/baselines/CodeLlama_13b_Instruct_hf.json:4]: maybe 1 epoch?

Comment (MohitIntel) [reply tests/baselines/CodeLlama_13b_Instruct_hf.json]: Fixed num_epochs and batchsize as 48(max supported).

File: tests/test_examples.py [hunk 2]
```diff
@@ -175,13 +175,14 @@ def to_test(model_name: str, multi_card: bool, deepspeed: bool, example_name: st
...
             "bigscience/bloom-7b1",
+            "codellama/CodeLlama-13b-Instruct-hf",
         ]
 
         if fsdp and os.environ.get("GAUDI2_CI", "0") == "0":
             return False
         elif model_name not in models_with_specific_rules and not deepspeed:
             return True
-        elif model_name == "gpt2-xl" and deepspeed:
+        elif model_name == "gpt2-xl" or model_name == "codellama/CodeLlama-13b-Instruct-hf" and deepspeed:
```
Comment (regisss) [inline tests/test_examples.py:185]:
```suggestion
        elif model_name in ["gpt2-xl", "codellama/CodeLlama-13b-Instruct-hf"] and deepspeed:
```

**Review by regisss — COMMENTED** [review]
All these results were obtained with Synapse 1.15?

**Comment by MohitIntel** [comment]
> All these results were obtained with Synapse 1.15?

Yes. Synapse 1.15 fw and driver on Release docker 1.15.479.

**Review by regisss — COMMENTED** [review]
Both T5-small tests fail on my instances:
- DL1 instance with 1.15
- Gaudi2 from IDC  with 1.14

The throughput I get is a bit too low compared to the baselines.
@MohitIntel Do you manage to have the tests pass on a regular basis?

**Comment by MohitIntel** [comment]
> Both T5-small tests fail on my instances:
> 
> * DL1 instance with 1.15
> * Gaudi2 from IDC  with 1.14
> 
> The throughput I get is a bit too low compared to the baselines. @MohitIntel Do you manage to have the tests pass on a regular basis?

Yes. I get the tests to pass on a regular basis. The baseline numbers I chose is worst of many runs. All my runs were above or at par with the baseline numbers here.

**Review by regisss — APPROVED** [review]
Now they pass, merging!

**Pull request merged**